web/src/components/core/ValidationErrors.jsx

```diff
@@ -77,27 +60,22 @@ const ValidationErrors = ({ title = _("Errors"), errors }) => {
           <button
```

Comment:
Not related with your changes, but do you mind removing the `else` above and just returning the content? :innocent:  Less indentation and easy to read by just relaying on the early return when errors.length === 1.